codespeed/settings.py

```diff
@@ -78,6 +78,13 @@
                          #     ('myexe', '21df2423ra'),
                          #     ('myexe', 'L'),]
 
+COMPARISON_TAGS = None  # List of tag names which should be included in the executables list on
```

Comment:
`COMPARISON_COMMIT_TAGS`? Also, this should be added to the readme

Reply:
0837385772d9444e999a317fd393694f803d7b7e